packages/web-components/src/components/ic-text-field/ic-text-field.tsx

```diff
@@ -596,7 +609,7 @@ export class TextField {
             {!multiline ? (
               <input
                 id={inputId}
-                name={name}
+                name={this.name}
```

Comment:
is this change needed? isn't name already accessible from the object restructuring in the render method (line 474)?